lib/index.js

```diff
@@ -539,7 +539,7 @@ function _getPublicKeyForPermissionCheck(publicKey, options, callback) {
   if(typeof publicKey === 'string') {
     publicKey = {id: publicKey || ''};
   }
-  api.getPublicKey(null, publicKey, function(err, publicKey) {
+  api.getPublicKey(publicKey, null, function(err, publicKey) {
```

Comment:
What's the conclusion on all of this?  I've fixed the order.  Do we need to make any changes to getPublicKey to default to 'undefined'?  getPublicKeys already does this.